content/post/writing-a-book-remotely.md

```diff
@@ -0,0 +1,576 @@
+
+We were all super happy with the outcome of the release party and our progress. At that time we already had 500 readers and getting more and more attention.
+
+## Life After the 1.0 Release 🏗
+
+> A book is never finished; it's abandoned. (Gene Fowler)
+
+While still being euphoric about releasing finish the first version of this book within one year, we knew that our journey wouldn't stop here. Right after the release party, we all took some time off and focussed more on spreading the news about the release rather than planning what's next.
+
+Ever since the release of version 1.0 in August 2021 we see great adoption and based on readers' feedback and also created a Slack channel for the community of the eBook for Q&A and general communication.
+
+Both Spring Boot and AWS are more or less mature, at least those areas that we talk about: SQS, SES, RDS, web development, etc. Still, there's much going on for the library that we're using for integrating AWS services ([Spring Cloud AWS](https://awspring.io/)). Furthermore, we also learn even better best practices or different ways of doing things.
+
+Thanks to the agile nature of Leanpub and given the fact that we don't ship a physical book to our readers, we can still update the eBook post delivery. Leanpub has a feature to notify the readers about a new version that they can then download for free.
+
+While we had no strict roadmap in mind, we all agreed to keep this book up-to-date to avoid a stale book.
+
+Whenever there's a new update for our eBook we also send out a small release information email to our newsletter to give a short overview of the changes:
+
+![Stratospheric Post Release Mailing List Update](/img/stratospheric-ebook/stratospheric-post-release-mailing-list-update.png#center "Stratospheric Post Release Mailing List Update")
+
+On the monetary side of things: We don't get rich from selling this book. It's some additional pocket money for us in the range of $150-300 for every author each month.
+
+To increase the visibility of our eBook we listed it on Amazon Kindle. While Amazon's royalty rules are not that optimal compared to Leanpub, we primarily list on Amazon to get in front of more developers. Amazon is one of the biggest search interfaces after Google and YouTube.
+
+We thought about a printed eBook version but for now postponed this move as we think that our audience is anyway more interested to read technical books on their devices. However, (at least I) want to have a physical copy of this book in my bookshelf somewhere in the future.
+
+Based on the feedback we got so far from our readers, we also decided to build an online course on top of the eBook to provide some more in-depth coverage of various areas. Given my [existing experience with creating and launching technical online courses](https://progmot.com/post/creating-and-selling-my-first-online-course/), this will be a great opportunity to apply my lessons learned from my first courses.
```

Comment:
```suggestion
Based on the feedback received so far, we also decided to build an online course on top of the eBook to provide some more in-depth coverage of various areas. Given my [existing experience with creating and launching technical online courses](https://progmot.com/post/creating-and-selling-my-first-online-course/), this will be a great opportunity to apply my lessons learned from my first courses.
```